src/pages/ReportDetailsPage.tsx

```diff
@@ -805,17 +812,8 @@ function ReportDetailsPage({policy, report, route, reportMetadata}: ReportDetail
                 isChatReportArchived: isMoneyRequestReportArchived,
                 isChatIOUReportArchived,
             });
-        } else {
-            deleteMoneyRequest(
-                iouTransactionID,
-                requestParentReportAction,
-                duplicateTransactions,
-                duplicateTransactionViolations,
-                iouReport,
-                chatIOUReport,
-                isChatIOUReportArchived,
-                isSingleTransactionView,
-            );
+        } else if (iouTransactionID) {
+            deleteTransactions([iouTransactionID], duplicateTransactions, duplicateTransactionViolations, currentSearchHash, isSingleTransactionView);
```

Comment:
@ZhenjaHorbach Could these changes be causing https://github.com/Expensify/App/issues/73330 ?

I'm reverting these here https://github.com/Expensify/App/pull/73379/files and it seems to fix the issue